### PR TITLE
Create Tab UI

### LINKED ISSUE #20: Create the "tab" page UI
image

It does not need to be dynamic, it just should have dummy data. Remember to set the <NextHead>  with meta tags and such

Also dont have too many constants (for colors) in your styled-components and use em for spacing whenever you can

### PR DESCRIPTION
**Pull request by samarthdave**
Still some ways to go; Here's how it looks so far:

image

What's left (updated):
- background of the container
- 🎛️ filter by "**day**" (eg. Day 1, Day 2, etc.) & 🔨  filter by **Category**

Closes #20

### CODE REVIEW

**Comment by samarthdave** [comment]
Update: https://styled-components.com/docs/advanced
Looks like it can take in props and use them directly within the template string. Let's see how this goes. I might just end up putting the colors in the theme file for global use.

**Comment by samarthdave** [comment]
Here's what I've got so far. I can change it to light grey if we want but I experimented with light blue. Thoughts? Removed the "Getting Started" section since I was getting some duplication errors. Otherwise, I just need to change the purple color but idk what blue is in the photo. Should I just use Variable blue values?

image

**Comment by samarthdave** [comment]
Here's the updated version with the light grey. Personally, I think the light blue looks better though.

image

File: src/styles/theme.config.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ export class THEMES {
       theme_dark_green: "#22b9a5",
       theme_light_green: "#86eace",
       theme_light_blue: "#4fc2f8",
+      theme_lighter_blue: "#f5faff",
```
Comment (geooot) [inline src/styles/theme.config.tsx:44]: Ah shoot its too light, lets try this
```suggestion
      theme_lighter_blue: "#F8FAFD",
```

**Pull request merged**